packages/build/tests/helpers/normalize.js

```diff
@@ -67,6 +67,9 @@ const NORMALIZE_REGEXPS = [
   [/^ +$/gm, ''],
   // HTTP errors are shown differently in Node 8
   [/ \.\.\.:443/g, ''],
+  // List of available plugins from `plugins.json`.
+  // That list changes all the time, so we need to remove it.
+  [/(Available plugins)[^]*Loading plugins/m, '$1'],
```

Comment:
This is a rather imperfect solution. 
Basically if "Available plugins is shown", it means plugins have been fetched. But the list itself is not shown to keep snapshots stable.